.github/release_plan.md

```diff
@@ -1,109 +1,62 @@
-
-# Move open issues (Tuesday of VS Code endgame week, XXX XX)
-
-- [ ] Ask the team to move all open issues to next milestone or remove milestone and put back in triage. This allows endgame champ to have a better understanding of done state.
-- [ ] Schedule a bugbash if needed.
-
-# Release candidate (Friday of VS Code endgame week, XXX XX)
+- [ ] Work with the build champ to drive the build to green by fixing/disabling tests or pinging area owners to do so.
 
+# Release candidate (Friday of endgame week)
 -   [ ] Review [Component Governance](https://dev.azure.com/monacotools/Monaco/_componentGovernance/191876) (Click on "microsoft/vscode-jupyter" on that page) and resolve all High/Severe issues.
-
     -   [ ] Focus on resolving `Critical` and `High` priority issues as others will be addressed in the `debt` week.
     -   [ ] Manually add any repository dependencies (if you can't add manually, refer [here](https://docs.opensource.microsoft.com/tools/cg/features/cgmanifest/)). Only add a cgmanifest.json if the components are not NPM or are not dev only.
         Instructions on updating `npm` dependencies in `package.json` & `package-lock.json` can be found [here](https://github.com/microsoft/vscode-jupyter/wiki/Resolving-Component-Governance-and-Dependabot-issues-(updating-package-lock.json)).
--   [ ] Create new release branch with format `release/release-YYYY.MM.100`
-    -   [ ] Create a pull request against `release/release-YYYY.MM.100` for changes
-    -   [ ] Bump the vscode engine version in package.json
-    -   [ ] Run `npm install` to verify `package-lock.json` did not get updated aside from the vscode engine version.
-        -   If there are minor changes, they can be just be updated in the release branch and later ported back to main.
+-   [ ] Create new release branch with format `release/release-YYYY.MM.100`.
+    -   [ ] Prepare a PR to this branch with the following changes...
+    -   [ ] Remove the `-insiders` from the vscode engine version. Run `npm install` to update this value in `package-lock.json`.
     -   [ ] Update [`CHANGELOG.md`](https://github.com/Microsoft/vscode-jupyter/blob/main/CHANGELOG.md)
-        -   [ ] Run [`news`](https://github.com/Microsoft/vscode-jupyter/tree/main/news) (typically `python news --final --update CHANGELOG.md | code-insiders -`)
+        -   If needed, install news deps by running `python3 -m pip install --user -r news/requirements.txt`
+        -   [ ] Run [`news`](https://github.com/Microsoft/vscode-jupyter/tree/main/news) (`python news --final --update CHANGELOG.md | code-insiders -`)
         -   [ ] Update the date at the top to when the release is planned.
         -   [ ] Copy over the "Thanks" section from the previous release into the "Thanks" section for the new release
         -   [ ] Make sure the "Thanks" section is up-to-date (e.g. compare to versions in [`requirements.txt`](https://github.com/microsoft/vscode-jupyter/blob/main/requirements.txt))
```

Comment:
did you figure this relationship out?

Reply:
No, that's still an open question for me. I think that's probably an obsolete point?

Reply:
@DonJayamanne can you explain what this means? There is very little in requirements.txt and it doesn't seem applicable to the Thanks section